src/backend/app/Providers/AppServiceProvider.php

```diff
@@ -83,6 +84,10 @@ public function boot() {
                 EBike::RELATION_NAME => EBike::class,
             ]
         );
+
+        if (config('app.env') === 'production') {
+            URL::forceScheme('https');
+        }
```

Comment:
I wonder if this is too narrow, and could cause some problems down the line. It should not be required if the Trust proxies are set up correctly, see above.